README.md

```diff
@@ -3,58 +3,53 @@
   
   # SnapShots
  
- ### Browse Screen 
-
-![Simulator Screen Shot - iPhone X - 2019-09-13 at 13 43 06](https://user-images.githubusercontent.com/43800526/64897394-1deb8e00-d638-11e9-912a-2db10d0bc12d.png)
-
-
-  ### Store Screen  
+ 
+### Store Screen  
  
 ![Simulator Screen Shot - iPhone X - 2019-09-13 at 13 43 31](https://user-images.githubusercontent.com/43800526/64897369-06140a00-d638-11e9-8e65-22e07a1d391b.png)
 
 
-### Brand Screen 
+### New Arrivals 
 
-![Simulator Screen Shot - iPhone X - 2019-09-13 at 13 43 34](https://user-images.githubusercontent.com/43800526/64897341-f0064980-d637-11e9-88eb-cbdbe8dfa014.png)
+![Simulator Screen Shot - iPhone X - 2019-09-13 at 13 43 46](https://user-images.githubusercontent.com/43800526/64897309-d1a04e00-d637-11e9-8e3f-e0faa4190614.png)
 
 
-### New Arrivals 
 
-![Simulator Screen Shot - iPhone X - 2019-09-13 at 13 43 46](https://user-images.githubusercontent.com/43800526/64897309-d1a04e00-d637-11e9-8e3f-e0faa4190614.png)
+# Instructions to run the project 
 
 
-### Sale 
+```Git Clone the repository``` 
 
-![Simulator Screen Shot - iPhone X - 2019-09-13 at 13 43 50](https://user-images.githubusercontent.com/43800526/64897252-9ef65580-d637-11e9-8594-9b9c06664449.png)
+```yarn install``` 
 
+```cd ios``` 
 
+```pod install```
 
+```react-native run-ios (react-native run-ios)```
 
-# Instructions to run the project 
+# Technologies: 
 
-1. Git Clone the repository 
+```GraphCool```
 
-1. yarn install 
+```GraphQL```
```

Comment:
Do not need to add GraphQL as a separate technology. GraphCool is using GraphQL.